build.gradle

```diff
@@ -72,17 +72,15 @@ dependencies {
     implementation "com.squareup.okhttp3:okhttp-sse:${okhttpclientVersion}"
     implementation 'com.moandjiezana.toml:toml4j:0.7.2'
     // use the android version because we don't want java 8 stuff
-    implementation 'com.google.guava:guava:26.0-android'
-    implementation 'com.google.code.gson:gson:2.9.0'
-    implementation 'commons-io:commons-io:2.11.0'
+    // TODO: Do we really need to introduce guava?
```

Comment:
if you're willing to try removal as part of this pr, and see how much turns red, it brings in `com.google.common.base` which is ref'd everywhere in app code for Preconditions and Objects convenience helpers, doesn't look like it's required though as a transitive dep from other dependencies.